armi/nuclearDataIO/__init__.py

```diff
@@ -173,7 +173,10 @@ def _getGammaKeywords(cycle, suffix, xsID):
         elif xsID is not None:
             keywords = [xsID]
             if suffix not in [None, ""]:
-                keywords.append("-" + suffix)
+                if "-" in suffix:
```

Comment:
I had to do some head scratching to figure out what this was trying to do. Maybe
```
if not suffix.startswith("-"):
    suffix = "-" + suffix
```
would be clearer

Reply:
GH isnt letting me comment above your changes, but even though this is private, I think a short docstring or comment describing what this function is supposed to do would help a lot